integration_tests/integration/serviceProviderReferrals.spec.js

```diff
@@ -842,7 +842,9 @@ describe('Service provider referrals dashboard', () => {
       referral: {
         interventionId: accommodationIntervention.id,
         serviceCategoryId: serviceCategory.id,
```

Comment:
Can we remove `serviceCategoryId` now? Or is there still another section of the journey that relies on it?

Reply:
I'm modifying the PP EOSR in this commit, however because I had to touch on some shared libraries means I have to do a few small changes on the SP side - there is another ticket that is for SP EOSR changes.

Reply:
ah sorry, that does make sense. All good then, we can remove these when you've done the SP side.